native/selected-text-reader/src/macos.rs

```diff
@@ -34,146 +31,34 @@ extern "C" {
     ) -> CGEventRef;
     fn CGEventSetFlags(event: CGEventRef, flags: CGEventFlags);
     fn CGEventPost(tap: CGEventTapLocation, event: CGEventRef);
+    fn CGEventSetIntegerValueField(event: CGEventRef, field: u32, value: i64);
```

Comment:
I did get rid of some of the methods we could get context to prefer doing the same cmd + c everytime for simplicity and consistency. This also shaved 100ms of time off selected text grabbing for me.